charter.md

```diff
@@ -14,22 +14,33 @@ proxy. Moreover, it would not introduce potentially redundant or unnecessary rec
 an established request/response semantic that can set up and configure flows for different services.
 
 The primary goal of this working group is to develop mechanism(s) that allow configuring and concurrently running
-multiple proxied stream- and datagram-based flows inside an HTTPS connection. These mechanism(s) are collectively called
+multiple proxied stream- and datagram-based flows inside an HTTP connection. These mechanism(s) are collectively called
 MASQUE. The group will specify HTTP and/or HTTP/3 extensions to enable this functionality.
 
-The group will focus on a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
-Server-initiated services are out of scope. The working group will first deliver a protocol solution for UDP CONNECT and
-a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
-IP proxying.
+The group will focus on extensions for a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
+Server-initiated services are out of scope.
```

Comment:
I have a pretty clear idea in my mind of what this tries to accomplish, I'd rather we figure out a way to phrase this instead of striking it. Perhaps something like:
```suggestion
MASQUE is only chartered to work on services that the client requests from the server.
Services that the server initiates without any prompt from the client are out of scope.
```
I think that makes it clear that UDP listen is in scope while still scoping out real work items.

Reply:
Maybe this is a semantic problem. Do you mean "proxy-initiated services"?

Reply:
Indeed, I meant proxy (or more specifically UDP proxy as defined in RFC 9298 or IP proxy from that draft). We didn't have that term defined back when we wrote the first charter.
```suggestion
MASQUE is only chartered to work on services that the client requests from the proxy.
Services that the proxy initiates without any prompt from the client are out of scope.
```

Reply:
WFM